docs/koji.md

```diff
@@ -0,0 +1,313 @@
+NDA3MDgwOTM0MzBaFw0yNDA3MDUwOTM0MzBaMGIxCzAJBgNVBAYTAlVTMQswCQYD
+VQQIEwJOQzEQMA4GA1UEBxMHUmFsZWlnaDEVMBMGA1UEChMMS2F0ZWxsbyBLb2pp
+MQswCQYDVQQLEwJDQTEQMA4GA1UEAxMHa29qaSBDQTCCASIwDQYJKoZIhvcNAQEB
+BQADggEPADCCAQoCggEBAMvihJFUzcwvIjip+ACWYoVvOlAecSZnox1FSezBPiWm
+280FrYf2wN+2s/FWvlDqUheAX03/DcZBPjHiB43v8AJwF3VNCGFu3CaqkQsg1T5s
+vw6pvmL0D5gj/wy25OnSOQIGN1EPnb+paCSCN7BRN5DJTk/jDX4ebIbhKFWgC6IH
+Jg3jmWvf3ROHXkSumMNMvLtCC91/Y6a28zBu91tL6uCTYBp7G/rVq0hG0doCMrrb
+gShVuNq2uoPvQXasjE54Nsd4PnkFbcX+pQcDnhlORBHTVk/m98t+ArOnWBKKQlQh
+88jPpVso/jKy6h8R3YHUZY3QY4G3lB7qPJRCI4SaRf8CAwEAAaOBxzCBxDAdBgNV
+HQ4EFgQUWaJRvcA/+GaPABgzU0wydC1WDlgwgZQGA1UdIwSBjDCBiYAUWaJRvcA/
++GaPABgzU0wydC1WDlihZqRkMGIxCzAJBgNVBAYTAlVTMQswCQYDVQQIEwJOQzEQ
+MA4GA1UEBxMHUmFsZWlnaDEVMBMGA1UEChMMS2F0ZWxsbyBLb2ppMQswCQYDVQQL
+EwJDQTEQMA4GA1UEAxMHa29qaSBDQYIJAJ/i4oLhK2xQMAwGA1UdEwQFMAMBAf8w
+DQYJKoZIhvcNAQEFBQADggEBAItZgucwDLFJI5U9ejzJXx6yw61/qlkqwNpN5608
+PCuz4zdiIH6Ju8dCaOdnWF19aQdASUxc2xjyRs2X6qqZvaX9j4lZXUHxXrGpXTiu
+BUJ4OsM3oDbyOvwj3EhK7+rSzjI3Cwx0V2n4xpnCLmfExlMt3Z/LreNUY4bKws0u
+nRImQExCmyy+ubY5kGUPEOp8fLhoXVpNyWUX0eTjbiWeqwiFIBgpZrLYFnq0SVCF
+lriG9rF2DdvfJFAxZOvbed3EFV/9FrEHhUD8cRdZZMteUh/LzNJZ7JVXTi5lC0Bz
+fMKMQGtd6DocgHLpv+5uJg8rAz8bAA3TmnHHk0SCu0iI+tw=
+-----END CERTIFICATE-----
+</pre>
+
+### Set up a kkoji alias
+
+Using the "kkoji" section created above in the koji config file, aliasing "kkoji" to the koji binary will cause it to use this section.
+
+    mkdir ~/bin
+ln -s /usr/bin/koji ~/bin/kkoji
+
+If you have an old version of the koji client, you may need to use a bash alias instead.  (Symlink approach is preferable as it works with xargs.)
+
+### Login to Koji WebUI
+
+You do not need to be logged to WebUI. But it is useful if you want to cancel or resubmit task. Or set up notifications. Run:
+
+<pre>
+openssl pkcs12 -export -inkey ~/.katello.cert -in ~/.katello.cert -CAfile ~/.katello-ca.cert -out katello_browser_cert.p12
+</pre>
+
+When prompted for a password, either leave it blank or specify one which is used to secure the output file - you then need to supply it again on import.
+
+And in Firefox do: {{{ Edit -> Preferences -> Advanced -> Encryption -> View Certificates -> Your Certificates -> Import... }}}
+
+### Repos
+
+Repositories are generated automatically, to manually run it use '''/usr/local/bin/katello-mash-split.py''' on the koji box.
+
+Final repositories can be accessed at http://koji.katello.org/releases/
+
+And can be rsynced using:
+<pre>
+rsync -alHvr rsync://koji.katello.org/releases /tmp/katello
+</pre>
+
+Repos are now regulary (after each mashing) synced by lzap to http://fedorapeople.org/groups/katello/releases/ which is public faced. The cronjob script is:
+
+<pre>
+40 7,13,18,21 * * * ssh fedorapeople.org 'cd /project/katello/bin && sh download-repos-from-koji'
+</pre>
+
+If you want to get writte access to fedorahosted.org /project/katello directory, ask Fedora Infrastructure team for ACLs.
```

Comment:
Let's not encourage this. It should end up on yum.theforeman.org at some point.

Reply:
Agreed.  Access to repo destinations in general shouldn't be a part of this doc, imho.